docs/UserGuide.md

```diff
@@ -811,6 +814,8 @@ in the future.
 
 `addqn qn/QUESTION`
 
+> :memo: You must navigate to a lesson view through `viewlsn` before this command can be used.
```

Comment:
Perhaps changing to
"You must navigate to the tutorial lesson page through `viewlsn` before this command can be used"
would be clearer

Reply:
Thanks, I amended this!